src/rust/ensogl/lib/core/src/control/io/mouse.rs

```diff
@@ -34,7 +34,8 @@ pub struct EventDispatcher<T> {
 
 impl<T> EventDispatcher<T> {
     /// Adds a new callback.
-    pub fn add<F:FnMut(&T)+'static>(&self, f:F) -> callback::Handle {
+    pub fn add<F>(&self, f:F) -> callback::Handle
+    where for <'x> F : FnMut(&'x T) + 'static {
```

Comment:
please write `for<'x>` instead of `for <'x>`, as this is parametrization of for and we write it so in other places as well

Reply:
Actually this is no longer needed, and should be reverted.